.github/CODEOWNERS

```diff
@@ -529,6 +529,7 @@
 /core/templates/**/*Spec.ts @marianazangrossi
 /extensions/**/*.spec.ts @marianazangrossi
 /extensions/**/*Spec.ts @marianazangrossi
+/core/templates/components/common-layout-directives/common-elements/loading-dots.component.spec.ts @marianazangrossi
```

Comment:
The lines above are already "covering" this file in order to me be its codeowner

Reply:
@marianazangrossi True, but for some reason, I kept getting the error that the spec file is not listed in CODEOWNERS, so I was not able to push. Can you think of any reason why this could be happening?

Reply:
I see, I think the changes you need to made is on line 194:
`/core/templates/components/common-layout-directives/common-elements/loading-dots.component*.ts @ankita240796`